test/verifiedTests/testTopology/testFindExtremePathway.m

```diff
@@ -46,14 +42,14 @@
     solverOK = changeCobraSolver(solverPkgs{k}, 'LP', 0);
 
     if solverOK == 1
-        v = findExtremePathway(model);
-        assert(all(abs(model.S * v) < tol))
-
-        % testing findExtremePathway with different arguments
-        if isfield(model, 'revRxns')
-            model = rmfield(model, 'revRxns');
-        end
-
+%         v = findExtremePathway(model);
```

Comment:
I can see why the test on `revRxns` is obsolete, but why is the test on ` v = findExtremePathway(model);`?

As `revRxns` is obsolete, I would recommend removing the test code ...